content/docs/self-hosted-runners.md

```diff
@@ -130,44 +158,51 @@ or GitLab
 the latter also supports
 [external secrets](https://docs.gitlab.com/ee/ci/secrets) for added security.
 
-When invoking the `cml-runner` command, you will need use environment variables
-to provide the following secrets:
+⚠️ You will need to create a
+[personal access token (PAT)](#personal-access-token) with repository read/write
+access and workflow privileges. In the example workflow above, this token is
+stored as `PERSONAL_ACCESS_TOKEN`.
 
-- A personal access token to register and remove self-hosted runners from your
-  repositories
-- Cloud credentials to create and destroy cloud resources, only when used in
-  conjunction with the `--cloud` option
+🛈 If using the `--cloud` option, you will also need to provide access
+credentials for your cloud compute resources as secrets. In the above example,
+`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` (with privileges to create &
+destroy EC2 instances) are required.
 
-### Personal access token
+### Personal Access Token
 
 <details>
 
 #### GitHub
 
-You can either
-[create a personal access token](https://docs.github.com/en/github-ae@latest/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)
-with the `repo` scope or a
-[GitHub App](https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps)
-with **Repository permissions / Administration** write permissions (for
-repository-level runners), or **Organization permissions / Self-hosted runners**
-write permissions (for organization-level runners).
+Use either:
+
+- a
+  [personal access token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+  with the **repo** scope, or
+- a
+  [GitHub App](https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps)
+  with **Repository permissions / Administration** write permissions (for
+  repository-level runners), or **Organization permissions / Self-hosted
+  runners** write permissions (for organization-level runners).
 
 </details>
 
 <details>
 
 #### GitLab
 
-You can either
-[create a personal access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
-with the `api` scope or a
-[project access token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html)
-if your GitLab instance supports this feature; the latter will only work for
-project-level
-([specific](https://docs.gitlab.com/ee/ci/runners/runners_scope.html#specific-runners))
-runners, not for instance-level
-([shared](https://docs.gitlab.com/ee/ci/runners/runners_scope.html#shared-runners))
-runners.
+Use either:
+
+- a
+  [personal access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
+  with the **api** scope, or
+- a
+  [project access token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html).
+  This only works for project-level
```

Comment:
`import this`

What's “this”? Can we state more clearly that “this” refers to project access tokens?

Reply:
already in a bullet point, so clearly not referring to the previous bullet point?

Reply:
🤦🏼 🤐  Oops! I missed the bullet point.